app/services/articles/feed.rb

```diff
@@ -43,5 +43,22 @@ def default_home_feed(user_signed_in: false)
       _featured_story, stories = default_home_feed_and_featured_story(user_signed_in: user_signed_in)
       stories
     end
+
+    def optimized_feed(user, test_variant)
+      # test_variants as defined in field_test.yml
+      # Each variant is it's own algorithm which defines all the branching scenarios.
+      case test_variant
+      when "base"
+        Article.limit(20)
```

Comment:
This is placeholder....

Each version should be different and contain its own hypotheses and world views—perhaps with some shared code, but each can be as distinct as we'd like.

Since this logic could be so delightfully isolated, it would be great if we could open up to the community opportunities to construct new feeds like this so it would be great if it were really easy to point people to the folder where to do all this

I'm excited for this page in docs, it should finally be straightfoward to explain how to help with the feed, and if somebody comes up with an option which could be better, we can merge and test.